apis/r/src/Makevars.in

```diff
@@ -1,15 +1,29 @@
 CXX_STD = CXX17
 
 ## We need the TileDB Headers, and for macOS aka Darwin need to set minimum version 10.14 for macOS
-PKG_CPPFLAGS = -I. -I../inst/include/ @tiledb_include@ -DR_BUILD @cxx17_macos@ -Iinclude
+#PKG_CPPFLAGS = -I. -I../inst/include/ @tiledb_include@ -DR_BUILD @cxx17_macos@ -Iinclude
+PKG_CPPFLAGS = -I. -I../inst/include/ @tiledb_include@ @cxx17_macos@ -Iinclude
 
 ## We also need the TileDB library
-PKG_LIBS = @cxx17_macos@ @tiledb_libs@ @tiledb_rpath@ lib/libtiledbsoma.a
+#PKG_LIBS = @cxx17_macos@ @tiledb_libs@ @tiledb_rpath@ lib/libtiledbsoma.a
+PKG_LIBS = @cxx17_macos@ -ltiledbsoma @tiledb_libs@ @tiledb_rpath@
 
 all: $(SHLIB)
-        # if we are
+        # If we are:
         #  - not on Window NT (a tip from data.table)
         #  - on macOS aka Darwin which needs this
         #  - the library is present (implying non-system library use)
         # then let us call install_name_tool
-	if [ "$(OS)" != "Windows_NT" ] && [ `uname -s` = 'Darwin' ] && [ -f ../inst/tiledb/lib/libtiledb.dylib ] && [ -f tiledbsoma.so ]; then install_name_tool -change libz.1.dylib @rpath/libz.1.dylib ../inst/tiledb/lib/libtiledb.dylib; install_name_tool -add_rpath @loader_path/../tiledb/lib tiledbsoma.so; fi
+	#
+	# Pro-tip: here you can run things like `otool -L tiledbsoma.so`, `ls ../inst`, etc. but
+	# the output will only be visible if the R build fails.
```

Comment:
I don't think that is entirely correct.  These line will alway run when the `if` tests above are true.  That includes CI for macOS, but also matching macOS runs outside CI.

Reply:
@eddelbuettel I mean the output being visible at the terminal when run interactively.